# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1302145605 25200
#      Wed Apr 06 20:06:45 2011 -0700
# Node ID c8be316f95438b7b3e8293c3ce0b6bba35fa6d3b
# Parent  2c4e52c83b64d04e14838f5f329161aabe844ee1
Group commonly used isXXX functions together in documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-04-06  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/numbers.txi, interpreter/strings.txi: Group commonly used
+	isXXX functions together in documentation.
+
 2011-04-06  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi, interpreter/var.txi: Move pack() function to
 	proper place in documentation.
 
 2011-04-06  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/diffeq.txi, interpreter/eval.txi, interpreter/plot.txi,
diff --git a/doc/interpreter/numbers.txi b/doc/interpreter/numbers.txi
--- a/doc/interpreter/numbers.txi
+++ b/doc/interpreter/numbers.txi
@@ -804,19 +804,19 @@ endfunction
 @end group
 @end example
 
 The following functions are available for determining the type of a
 variable.
 
 @DOCSTRING(isnumeric)
 
-@DOCSTRING(isreal)
+@DOCSTRING(isfloat)
 
-@DOCSTRING(isfloat)
+@DOCSTRING(isreal)
 
 @DOCSTRING(iscomplex)
 
 @DOCSTRING(ismatrix)
 
 @DOCSTRING(isvector)
 
 @DOCSTRING(isrow)
diff --git a/doc/interpreter/strings.txi b/doc/interpreter/strings.txi
--- a/doc/interpreter/strings.txi
+++ b/doc/interpreter/strings.txi
@@ -512,31 +512,31 @@ isalpha ("!Q@@WERT^Y&")
      @result{} [ 0, 1, 0, 1, 1, 1, 1, 0, 1, 0 ]
 @end group
 @end example
 
 @DOCSTRING(isalnum)
 
 @DOCSTRING(isalpha)
 
-@DOCSTRING(isascii)
-
-@DOCSTRING(iscntrl)
-
-@DOCSTRING(isdigit)
-
-@DOCSTRING(isgraph)
-
 @DOCSTRING(isletter)
 
 @DOCSTRING(islower)
 
-@DOCSTRING(isprint)
+@DOCSTRING(isupper)
+
+@DOCSTRING(isdigit)
+
+@DOCSTRING(isxdigit)
 
 @DOCSTRING(ispunct)
 
 @DOCSTRING(isspace)
 
-@DOCSTRING(isupper)
+@DOCSTRING(iscntrl)
+
+@DOCSTRING(isgraph)
 
-@DOCSTRING(isxdigit)
+@DOCSTRING(isprint)
+
+@DOCSTRING(isascii)
 
 @DOCSTRING(isstrprop)
